packages/aragon-wrapper/src/index.js

```diff
@@ -69,7 +69,10 @@ export const setupTemplates = (
 export default class Aragon {
   constructor (daoAddress, options = {}) {
     const defaultOptions = {
-      provider: detectProvider()
+      provider: detectProvider(),
+      defaultGasPriceFn: () => {
```

Comment:
We should add this new option to the documentation of the constructor.

Reply:
Yes. I wanted to wait and see if you're good with `defaultGasPriceFn` or if you prefer a different name.